Lib/distutils/sysconfig.py

```diff
@@ -456,7 +456,8 @@ def _init_nt():
         g['EXT_SUFFIX'] = _imp.extension_suffixes()[0]
     else:
         plat_tag = get_platform().replace('-', '_')
-        g['EXT_SUFFIX'] = '.cp{0.major}{0.minor}-{1}.pyd'.format(sys.version_info, plat_tag)
+        host_plat_tag = get_host_platform().replace('-', '_')
+        g['EXT_SUFFIX'] = _imp.extension_suffixes()[0].replace(host_plat_tag, plat_tag)
```

Comment:
If it's a debug build it also needs and _d

release:
>>> from distutils.sysconfig import get_config_var
>>> get_config_var('EXT_SUFFIX')
'.cp38-win32.pyd'

debug:
>>> from distutils.sysconfig import get_config_var
>>> get_config_var('EXT_SUFFIX')
'**_d**.cp38-win32.pyd'

Reply:
Or maybe this doesn't matter.  What I remember (from 3 months ago) is that building a debug .whl didn't help.
Also I can't seem to construct a command line to build a debug .whl without having to manually rename it after.

Reply:
The wheel tag format is different from the import tag format, and you need to align a lot of planets to get it to work.

The bit that's bothering me most here right now is that this is an initialization function that gets run on import. I'd rather split this block into its own `util.get_ext_suffix()` function that can handle changes at runtime.

Reply:
I have removed these changes.